### PR TITLE
Move to Monorepo

### PR DESCRIPTION
**Pull request by mawburn**
Also updated some of the meta data and what not.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,49 +1,3 @@
...
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+Mono repo for Portaler - the best mapping tool for Albion Online in the history of the world.
```
Comment (Tebro) [inline README.md:3]: Maybe a bit less neutral wording? Let the thing speak for itself :D

File: packages/frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "lint:fix": "eslint --fix --ext .js,.jsx,.ts,.tsx --ignore-pattern '/node_modules/' ./src"
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "^4.0.0-alpha.56",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/cytoscape": "^3.14.7",
```
Comment (Tebro) [inline packages/frontend/package.json:21]: Would be good to have `@types` as devDependencies

File: packages/main-api/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# portalter-server
```
Comment (Tebro) [inline packages/main-api/README.md:1]: Typo

File: packages/main-api/server/utils.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+// async/await error catcher
+const catchAsyncErrors = (fn) => (req, res, next) => {
+  const routePromise = fn(req, res, next);
+  if (routePromise.catch) {
```
Comment (Tebro) [inline packages/main-api/server/utils.js:4]: catch is always present on a promise, and any async function returns a promise. This check is not needed. If this is to make sure devs don't make mistakes then a typing system does it way better.

**Review by Tebro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mawburn) [reply README.md:3]: Haha good point, I did that as a joke on stream.

File: packages/frontend/package.json (hunk 2 above)
Comment (mawburn) [reply packages/frontend/package.json:21]: I'm always conflicted where to put them.

File: packages/main-api/server/utils.js (hunk 4 above)
Comment (mawburn) [reply packages/main-api/server/utils.js:4]: I wouldn't worry about this for now, this is just a placeholder. I'm going to do a live code review with him @mmcken3, I just moved his stuff into the repo.

**Pull request merged**